fuel/converters/adult.py

```diff
@@ -0,0 +1,140 @@
+import os
+
+import h5py
+import numpy
+
+from fuel.converters.base import fill_hdf5_file
+
+
+def convert_to_one_hot(y):
+    """
+    converts y into one hot reprsentation.
+
+    Parameters
+    ----------
+    y : list
+        A list containing continous integer values.
+
+    Returns
+    -------
+    one_hot : numpy.ndarray
+        A numpy.ndarray object, which is one-hot representation of y.
+
+    """
+    max_value = max(y)
+    min_value = min(y)
+    length = len(y)
+    one_hot = numpy.zeros((length, (max_value - min_value + 1)))
+    one_hot[numpy.arange(length), y] = 1
+    return one_hot
+
+
+def convert_adult(directory, output_directory,
+                  output_filename='adult.hdf5'):
+    """
+    Convert the Adult dataset to HDF5.
+
+    Converts the Adult dataset to an HDF5 dataset compatible with
+    :class:`fuel.datasets.Adult`. The converted dataset is saved as
+    'adult.hdf5'.
+    This method assumes the existence of the file `adult.data` and
+    `adult.test`.
+
+    Parameters
+    ----------
+    directory : str
+        Directory in which input files reside.
+    output_directory : str
+        Directory in which to save the converted dataset.
+    output_filename : str, optional
+        Name of the saved dataset. Defaults to `adult.hdf5`.
+
+    Returns
+    -------
+    output_paths : tuple of str
+        Single-element tuple containing the path to the converted dataset.
+
+    """
+    # classes = {b'>50K': 0, b'<50K': 1}
```

Comment:
It's providing basic information about the input and output of dataset, though not very informative. What do you suggest to add there?